pubsub/redis/redis.go

```diff
@@ -260,7 +260,7 @@ func (r *redisStreams) worker() {
 // by `reclaimPendingMessagesLoop`.
 func (r *redisStreams) processMessage(msg redisMessageWrapper) error {
 	r.logger.Debugf("Processing Redis message %s", msg.messageID)
-	if err := msg.handler(&msg.message); err != nil {
+	if err := msg.handler(context.Background(), &msg.message); err != nil {
```

Comment:
Here you can use `r.ctx` instead of `context.Background()`.

Reply:
Done 👍